uiplib/settings.py

```diff
@@ -13,18 +13,18 @@
 class ParseSettings:
     """Argument Parser class."""
 
-    def __init__(self):
+    def __init__(self, args=None):
         """Initialize the argument parser."""
         self.settings = self.get_settings_from_file()
-        self.settings.update(self.get_settings_from_cli())
+        self.settings.update(self.get_settings_from_cli(args))
 
     def get_settings_from_file(self):
         """Return the settings from file."""
         with open(settings_file_path, "r") as settings_file:
             settings = json.loads(settings_file.read())
         return settings
 
-    def get_settings_from_cli(self):
+    def get_settings_from_cli(self, args=None):
```

Comment:
this is not the right way to tests infact these tests are meaningless as they dont test the functionality we want them to test. Instead they test the functionality you added just so you could test it